Client/src/i18n/ru/errors.js

```diff
@@ -1,5 +1,5 @@
 export default {
-  ServerError: "",
+  ServerError: "Ошибка на сервере",
```

Comment:
Do we need translation for this one?

Reply:
I dont understand. Any code need some translations to show on client.